pkg/utils/options/options.go

```diff
@@ -41,13 +42,14 @@ func MustParse() Options {
 
 // Options for running this binary
 type Options struct {
-	ClusterName     string
-	ClusterEndpoint string
-	MetricsPort     int
-	HealthProbePort int
-	WebhookPort     int
-	KubeClientQPS   int
-	KubeClientBurst int
+	ClusterName            string
+	ClusterEndpoint        string
+	MetricsPort            int
+	HealthProbePort        int
+	WebhookPort            int
+	KubeClientQPS          int
+	KubeClientBurst        int
+	DefaultInstanceProfile string
```

Comment:
This makes me think about neutrality as we add more vendor specific flags. I wonder if there's a way for us to register cloud provider flags into the binary somehow. We can leave this for another day.

Reply:
I thought about that too. But... will leave that one up to you all to figure out! :laughing: